### ISSUE TITLE
Extra spaces being inserted on lists

### ISSUE THREAD

**Issue by Frenchomatic** [ahmedkaludi/accelerated-mobile-pages#2397]
A bit confused by this one. 

I have a list in which I recolor bullet points using some fairly standard css code

```
li:before{
content: "•";
color: #75ad4a;
display: inline-block;
width: 0.7em;
margin-left: -0.7em;
font-size: 26px;
position: relative;
top: 3px;
}

ul {
list-style-type: none;
}
```

However,  I get an extra space being into rendered text on lists when you have opposing brackets  in the  html   `><`. 

For example if I put this in the editor `<li><strong>bla bla</strong> - some text</li>` 

This gets rendered in the source code as 

`<li>SPACE<strong>bla bla</strong> - some text</li>` 

but with no opposing brackets like below no space gets put in

`<li>bla bla - some text</li>`

If I remove the ::before css then the problem no longer occurs. I am wondering how the plugin handles the ::before. It is like it inserts a space when it shouldn't.

**Comment by Frenchomatic** [comment]
And I confirm it doesn't do this when not using AMP. The :before is handled correctly.

**Comment by Frenchomatic** [comment]
But this was recently broken - it wasn't  like that on previous versions. Please issue a bug fix.

**Comment by Frenchomatic** [comment]
And if I roll back to a previous version Version 0.9.96.1 for example this bug doesn't exist.

**Comment by Frenchomatic** [comment]
see #2406

**Comment by Zabi09** [comment]
This will be fixed in #2445
